### DISCUSSION TITLE
Excellent work on Pretty Google Calendar - thanks

### DISCUSSION THREAD

**Discussion by Rhythmick7** [community/community#110535] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm helping a small bowling club get their Wordpress site doing what they need. Which isn't much, but a club calendar is definitely needed. Seems to me much easier to create a google id and calendar then pull the data for it into this excellent tool, rather than try to maintain the data in the site. Discovered the plug-in, created the calendar API and got it all working with no fuss, about 30 minutes.

Question or maybe product suggestion :- is it possible to filter the calendar events shown ? So for example I could show a calendar for HOME events (when the green is in use) and another with ALL events. I'm thinking keep it simple by filtering on whether event title contains a keyword, for example.

Great work - I'll ask the club treasurer to send a little sponsorship payment to say thanks.

**Comment by queenofcorgis** [comment]
Hi @Rhythmick7 , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
